examples/src/components/GeoPushpinChartConfigurationPointsExample.jsx

```diff
@@ -0,0 +1,113 @@
+
+export class GeoPushpinChartConfigurationPointsExample extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            config: {
+                tooltipText: tooltipTextAttribute,
+                mapboxToken: MAPBOX_TOKEN,
+                points: {
+                    groupNearbyPoints: true,
+                },
+            },
+            customGroupNearbyPointsUsed: false,
+        };
+    }
+    onLoadingChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onLoadingChanged", ...params);
+    }
+
+    onError(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onError", ...params);
+    }
+
+    onZoomChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onZoomChanged", ...params);
+    }
+
+    onCenterPositionChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onCenterPositionChanged", ...params);
+    }
+
+    onCustomPointsSizeTabSelect = () => {
+        const { config: currentConfig } = this.state;
+        this.setState({
+            config: {
+                ...currentConfig,
+                points: {
+                    minSize: "0.5x",
+                    maxSize: "1.5x",
+                },
+            },
+            customGroupNearbyPointsUsed: false,
+        });
+    };
+
+    onCustomGroupNearbyPointsSelect = () => {
+        const { config: currentConfig } = this.state;
+        this.setState({
+            config: {
+                ...currentConfig,
+                points: {
+                    groupNearbyPoints: false,
+                },
+            },
+            customGroupNearbyPointsUsed: true,
+        });
+    };
+
+    render() {
+        const style = { height: "500px" };
+        const { config: geoConfig, customGroupNearbyPointsUsed } = this.state;
+        // eslint-disable-next-line no-console
+        return (
+            <div className="s-geo-chart">
+                <button className="s-change-points-size" onClick={this.onCustomPointsSizeTabSelect}>
+                    Custom Points Size
+                </button>
+
+                <button
+                    className="s-change-group-neaerby-points"
+                    onClick={this.onCustomGroupNearbyPointsSelect}
+                >
+                    Group nearby points
+                </button>
+
+                <div style={style} className="s-geo-pushpin-chart-configuration-custom-points">
+                    <GeoPushpinChart
+                        projectId={projectId}
+                        location={locationAttribute}
+                        size={customGroupNearbyPointsUsed ? null : sizeMeasure}
```

Comment:
Please add a comment here.
For ex: `groupNearbyPoints is supported in Geo chart which had location attribute only`

Reply:
fixed